e2e/tests/workspace/workspace-reader-actions.spec.ts

```diff
@@ -0,0 +1,49 @@
+import WorkspacesPage from 'app/page/workspaces-page';
+import { signIn } from 'utils/test-utils';
+import WorkspaceCard from 'app/component/workspace-card';
+import { config } from 'resources/workbench-config';
+import { LinkText, WorkspaceAccessLevel } from 'app/text-labels';
+import HomePage from 'app/page/home-page';
+import * as fp from 'lodash/fp';
+
+describe('Workspace READER actions tests', () => {
+  beforeEach(async () => {
+    await signIn(page, config.collaboratorUsername, config.userPassword);
+  });
+
+  // Tests don't require creating new workspace
+  test('READER cannot share, edit or delete workspace', async () => {
+    const homePage = new HomePage(page);
+    await homePage.getSeeAllWorkspacesLink().click();
+
+    const workspacesPage = new WorkspacesPage(page);
+    await workspacesPage.waitForLoad();
+
+    // Verify Workspace Access Level is READER.
+    const workspaceCards = await WorkspaceCard.findAllCards(page, WorkspaceAccessLevel.Reader);
+    if (workspaceCards.length === 0) {
+      return; // end test because no Workspace card available for checking
```

Comment:
Why not just create a new workspace if there aren't any READER ones?

Reply:
I want to keep number of new workspace as low as possible. Test runs faster and leaves no footprint (not changing application data in anyway). This type of tests can be selected to run ini prod env if we want to in future.

Reply:
I guess the tester users have access to a lot of workspaces already.